Add recap blog post for SQL training 2 (#563)
trinodb/trino.io: _posts/2023-11-01-sql-training-2.md

---
layout: post
title: "Advanced analytics with SQL and Trino"
author: "Manfred Moser"
excerpt_separator: <!--more-->
image: /assets/trino-sql.png
show_pagenav: false
---

In the second part of our training series [Learning SQL with Trino from the

> experts]({% post_url 2023-09-27-training-series %}) Martin Traverso and myself

Reviewer: ```suggestion
experts]({% post_url 2023-09-27-training-series %}) Martin Traverso and I
```

built on top of the foundational knowledge from the [first training session]({%
post_url 2023-10-18-sql-training-1 %}). We continued to learn more about data

> types and working with them, including the important strings, numeric, temporal

Reviewer: ```suggestion
types and working with them, including the important strings, numeric, temporal,
```

and JSON types.

<!--more-->

The recording of the event allows you to watch it all as if you attended live,
jump to specific sections as desired, or pause while you follow along with the
demos:

{% youtube S-mfueDmXds %}

Following are a couple of specific timestamps for interesting
topics snippets:

> * [Temporal data type](https://www.youtube.com/watch?v=S-mfueDmXds&t=601s)

Reviewer: ```suggestion
* [Temporal data types](https://www.youtube.com/watch?v=S-mfueDmXds&t=601s)
```

* [Strings](https://www.youtube.com/watch?v=S-mfueDmXds&t=1920s)

> * [Numeric](https://www.youtube.com/watch?v=S-mfueDmXds&t=2442s)

Reviewer: ```suggestion
* [Numeric types](https://www.youtube.com/watch?v=S-mfueDmXds&t=2442s)
```

* [URL parsing and more](https://www.youtube.com/watch?v=S-mfueDmXds&t=2705s)
* [JSON](https://www.youtube.com/watch?v=S-mfueDmXds&t=2850s)

The full timestamps for every part of the talk are in the description on
YouTube.

Also make sure you take advantage of these additional resources:

* [General overview slide deck for the
  series]({{site.baserurl}}/assets/blog/sql-training-series-starburst-2023.pdf),
  with links to resources like our [community
  chat]({{site.baserurl}}/slack.html)
* Slides deck for [Advanced analytics with SQL and
  Trino](https://trinodb.github.io/presentations/presentations/sql-adv-analytics/index.html),
  including a file with all SQL statements ready to go
* [Trino: The Definitive Guide]({{site.baserurl}}/trino-the-definitive-guide.html)

> We are half way through the series, and there is lots more to cover. Don't

Reviewer: ```suggestion
We are halfway through the series, and there is lots more to cover. Don't
```

> forget to register for the next ones, join us to ask specific questions, and

Reviewer: ```suggestion
forget to register for the next sessions, join us to ask specific questions, and
```
